client/lib/signup/cart.js

```diff
@@ -14,11 +14,13 @@ import productsListFactory from 'lib/products-list';
 const productsList = productsListFactory();
 import { cartItems, fillInAllCartItemAttributes } from 'lib/cart-values';
 
-function addProductsToCart( cart, newCartItems ) {
+function addProductsToCart( cart, newCartItems, tldLandingPageNonce ) {
```

Comment:
I feel like this approach is too broad and too specific at the same time. What if we want to add another property like this? IMHO, it's not `addProductsToCart`'s responsibility to know this. `tld_lp_nonce` should be set when adding the domain registration product to cart, not broadly to all products. Especially since it's only used by the domain registration one.